src/main/java/org/folio/inventory/support/http/server/JsonResponse.java

```diff
@@ -30,14 +31,14 @@ public static void success(HttpServerResponse response,
     response(response, body, 200);
   }
 
-  public static void successWithEmptyBody(HttpServerResponse response) {
-    emptyResponse(response, 201);
+  public static void successWithEmptyIds(HttpServerResponse response) {
+    successWithIds(response, new ArrayList<>());
   }
 
   public static void successWithIds(HttpServerResponse response, List<String> ids) {
     JsonObject nonUpdatedIds = new JsonObject();
     nonUpdatedIds.put("nonUpdatedIds", ids);
-    response(response, nonUpdatedIds, 201);
+    response(response, nonUpdatedIds, 200);
   }
```

Comment:
Should we move these methods to some separate class or to MoveResource? As far as I can see, this `JsonResponse` contains common responses, that can be reused, but `successWithIds` is a specific API response.